docs/predictors/setup-storage.md

```diff
@@ -1,14 +1,16 @@
 # Set up Storage for Loading Models
 
-You will need access to an S3-compatible object storage, for example [MinIO](https://github.com/minio/minio). To provide access to the object storage, use the `storage-config` secret.
+You will need access to an S3-compatible object storage, for example [MinIO](https://github.com/minio/minio). To configure access to the object storage, use the `storage-config` secret.
 
-## Deploy a model from your own object storage
+Alternatively, models can be stored on a Kubernetes Persistent Volume. Persistent Volume Claims can either be pre-configured in the `storage-config` secret, or, the `allowAnyPVC` config flag can be enabled, so that any PVC can be mounted dynamically at the time a predictor or inference service is deployed.
```

Comment:
```suggestion
Alternatively, models can be stored on a Kubernetes Persistent Volume. Persistent Volume Claims can either be pre-configured in the `storage-config` secret, or, the `allowAnyPVC` configuration flag can be enabled, so that any PVC can be mounted dynamically at the time a predictor or inference service is deployed.
```